agenta-backend/agenta_backend/routers/app_router.py

```diff
@@ -284,6 +286,51 @@ async def create_app(
         raise HTTPException(status_code=500, detail=str(e))
 
 
+@router.put("/{app_id}/", response_model=UpdateAppOutput, operation_id="update_app")
```

Comment:
I know that we haven't had a discussion about this but a "partially update of some attributes of a resource" is more a `PATCH` than a `PUT`. But this is non-blocking at the moment.

Reply:
I chose to make it a PUT request with the intention that new attributes may be added in the future that would need updating at some point.

Reply:
Ah, got it. Thanks.